README.md

```diff
@@ -1,8 +1,8 @@
 # OK
 
-**Elegant error handling in elixir pipelines. See [Handling Errors in Elixir](http://insights.workshop14.io/2015/10/18/handling-errors-in-elixir-no-one-say-monad.html) for a more detailed explanation**
+**Elegant handling of idiomatic Erlang conventions of `:ok`/`:error` tuples in Elixir. This includes more concise and readable `with` statement syntax, a tagged-enabled pipeline operator, and semantic pattern matching.**
```

Comment:
This is a too complicated intro. semantic pattern matching is a term I made up and tag-enabled pipeline is missleading as it only works with one kind of tab. I've tried to just say what it does.

Result monad in Elixir, for elegant error handling